CMakeLists.txt

```diff
@@ -104,7 +104,9 @@ message("Debug flags:")
 message("  CMAKE_C_FLAGS_DEBUG                     : ${CMAKE_C_FLAGS_DEBUG}")
 message("  CMAKE_CXX_FLAGS_DEBUG                   : ${CMAKE_CXX_FLAGS_DEBUG}")
 message("Install:")
-message("  CMAKE_INSTALL_RREFIX                    : ${CMAKE_INSTALL_RREFIX}")
+message("  CMAKE_INSTALL_RREFIX                    : ${CMAKE_INSTALL_PREFIX}")
```

Comment:
I believe it should be PREFIX :)

Reply:
arggh. I did not check the spelling on the left side of the expression... Thank you for the catch.